Formula/v/vunnel.rb

```diff
@@ -122,6 +117,11 @@ class Vunnel < Formula
     sha256 "942c5a758f98d790eaed1a29cb6eefc7ffb0d1cf7af05c3d2791656dbd6ad1e1"
   end
 
+  resource "ruff" do
+    url "https://files.pythonhosted.org/packages/5c/76/f1fbd0a6625d1f3044854b540642171acd02bd91c592001aac9f9606474c/ruff-0.1.14.tar.gz"
+    sha256 "ad3f8088b2dfd884820289a06ab718cde7d38b94972212cc4ba90d5fbc9955f3"
+  end
+
```

Comment:
Can we use our `ruff` formula here?

Reply:
Let's find out!  Built locally on my machine without an issue.